Docker/gen.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -e
+
+# number of primary+worker instances to start.
+num=$1
+
+if [ -z "${num}" ]; then
+    echo usage: $0 number_of_instances
+    exit 1
+fi
+
+if [ ! -x ../target/release/node ]; then
+    echo no narwhal node command found.
+    exit 1
+fi
```

Comment:
The script assumes that the code has been compiled and the `node` binary is available. I would suggest to not assume that the user knows about it - or what are the steps they need to take. So I suggest the following:
1. Add a suggestion on the above message (e.x node binary not found under `target/release/node`. Please run `cargo build --release --all-features` first and try again)
2. Or try to automatically compile the code (and let the user know about with a message)
3. Or add a section to README that clearly explains that `node` binary should be available and compiled first with the command `cargo build --release --all-features`

Reply:
I'll add this as a prerequisite in the README.md.